llama_index/embeddings/azure_openai.py

```diff
@@ -24,6 +25,7 @@ class AzureOpenAIEmbedding(OpenAIEmbedding):
 
     _client: AzureOpenAI = PrivateAttr()
     _aclient: AsyncAzureOpenAI = PrivateAttr()
+    _http_client: Optional[httpx.Client] = PrivateAttr()
```

Comment:
since `OpenAI` class has this already, and azure extends it, no need to redefine it here